lib/archival_storage_ingest/exception/ingest_exception.rb

```diff
@@ -1,4 +1,8 @@
 # frozen_string_literal: true
 
 class IngestException < StandardError
+  DEFAULT_ERROR_MESSAGE = 'Automated ingest encountered error'
+  def initialize(msg = :DEFAULT_ERROR_MESSAGE)
```

Comment:
I don't think the `:` is needed (or wanted). I think that sets the default error message to the _symbol_ `:DEFAULT_ERROR_MESSAGE`, not the value of the constant `DEFAULT_ERROR_MESSAGE`.